dynamic_movement_primitive/StateMsg internal
dynamic_movement_primitive/StateMsg target
dynamic_movement_primitive/StateMsg current

float64 start
float64 goal

float64 f
float64 ft

# float64[] function_input
# float64[] function_target
